src/coreclr/tools/aot/ILCompiler.Compiler.Tests/ILCompiler.Compiler.Tests.Assets/ILCompiler.Compiler.Tests.Assets.csproj

```diff
@@ -8,6 +8,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <!-- Don't add references to the netstandard platform since this is a core assembly -->
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <Features>noRefSafetyRulesAttribute=true</Features>
```

Comment:
What does this do?

Reply:
This disables the compiler emitting the `[assembly: RefSafetyRules]` attribute + supporting type. The core attribute types are incorrectly defined in this project and by default results in a compilation error. Given this change is being back ported to `release/7.0` I opted for disabling the attribute emit vs. churning the code further. @cston and I were going to follow up with a PR in a bit that flips this off, fixes the attributes and let that play out separately.